app/code/Magento/Checkout/Controller/Sidebar/UpdateItemQty.php

```diff
@@ -55,7 +55,7 @@ public function __construct(
     public function execute()
     {
         $itemId = (int)$this->getRequest()->getParam('item_id');
-        $itemQty = (int)$this->getRequest()->getParam('item_qty');
+        $itemQty = (float)$this->getRequest()->getParam('item_qty');
```

Comment:
This change is related with issues or PR?

Reply:
No
Thanks

Reply:
Can you delete it? To focus only in the issue. Thanks!

Reply:
I have reverted.
Thanks